changelog/unreleased/issue-2241

```diff
@@ -0,0 +1,10 @@
+Bugfix: Hide URL passwords from logs
+
+When using a password in the URL of a rest backend, in some cases
+the password was displayed in the restic log:
+- when creating a repository
+- when restic failed to open the repository
+
+The URL is now displayed with "***" in place of the password.
```

Comment:
Since there is no "log" in restic, only its output, and the changelog generator restic uses doesn't support lists, I think it's better to formulate this as follows:

~~~
Bugfix: Hide password in REST URLs

When using a password in the REST backend repository URL,
the password could in some cases be included in the output
from restic, e.g. when initializing a repo or during an error.

The password is now replaced with "***" where applicable.
~~~